Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.yml

```diff
@@ -29,16 +41,16 @@ configuration:
   name: incidentType
   required: false
   type: 13
-- defaultvalue: 1 hours
-  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
   name: fetchTime
   required: false
   type: 0
-- defaultvalue: '20'
-  display: Fetch limit (minimum is 20)
+  defaultvalue: 1 hours
```

Comment:
was this on purpose?

Reply:
if you look closely, only the order of keys was changed and github makes it look like values were changed (I double checked this)